src/main/java/seedu/address/logic/commands/util/EditElderlyDescriptor.java

```diff
@@ -23,11 +23,15 @@
 public class EditElderlyDescriptor extends EditPersonDescriptor {
     private RiskLevel riskLevel;
 
+    /**
+     * Default empty constructor.
```

Comment:
Let's start the javadocs with a verb
```suggestion
     * Constructs a default empty descriptor.
```